### PR TITLE
Added carousel functionality to related props and am rendering with data from API. Included CSS.

### PR DESCRIPTION
**Pull request by EmmaHelser**

### CODE REVIEW

File: server/controller/RelatedProductsController.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 const relatedProductsModel = require('../model/RelatedProductsModel.js');
-const token = require('../../token.js')
+const tokenObj = require('../../token.js')
```
Comment (BrandonParker) [inline server/controller/RelatedProductsController.js:2]: nice change

File: client/src/modules/RelatedProducts/RelatedProducts.jsx [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
+    this.state = {
+      compare: false,
+      compareProduct: {}
+    }
+  }
+
+  render () {
+    return (
+      <div className='carosel'>
+        {this.props.relatedInfo.start !== 0 ? <button className='previous nav' onClick={this.props.caroselClickLeft}>&larr;</button> : null}
```
Comment (BrandonParker) [inline client/src/modules/RelatedProducts/RelatedProducts.jsx:16]: Good use of html for the left arrow, if relatedInfo.start is equal to 0, why do you want to set it equal to null? Does that signify no related products?

File: client/src/modules/RelatedProducts/RelatedProductsYourOutfit.jsx [hunk 3]
```diff
@@ -16,6 +23,26 @@ class Related extends React.Component {
...
+  caroselClickLeft () {
+    const start = this.state.start - 1;
+    const end = this.state.end - 1;
+    const carosel = this.state.relatedProductsList.slice(start, end);
+
+    console.log(carosel);
+
+    this.setState({relatedCarosel: carosel, start: start, end: end});
+  }
+
```
Comment (BrandonParker) [inline client/src/modules/RelatedProducts/RelatedProductsYourOutfit.jsx:45]: Your solution looks simple and effective.

**Review by BrandonParker — APPROVED** [review]
Overall it looks good

File: client/src/modules/RelatedProducts/RelatedProducts.jsx (hunk 2 above)
Comment (EmmaHelser) [reply client/src/modules/RelatedProducts/RelatedProducts.jsx:16]: Null will make the button disappear when there are no more related products to load and view in the carousel.

**Pull request merged**